metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -527,6 +534,32 @@ object MetalsEnrichments
 
   }
 
+  implicit class XtensionURI(value: URI) {
+    def toAbsolutePath: AbsolutePath = toAbsolutePath(followSymlink = true)
+    def toAbsolutePath(followSymlink: Boolean): AbsolutePath = {
+      val path =
+        if (value.getScheme() == "jar")
+          Try {
+            AbsolutePath(Paths.get(value))
+          } match {
+            case Success(path) => path
+            case Failure(_) =>
+              // don't close - put up with the resource staying open so all AbsolutePath methods don't have to be wrapped
+              m.internal.io.PlatformFileIO.newFileSystem(
+                value,
+                new java.util.HashMap[String, String]()
```

Comment:
```suggestion
                create = false
```
? 

Not 100% sure what create does, but we sue it in a lot of places. I asume it means it will not throw an exception if the jar system is already created?

Reply:
I think `create` is there if you want to create a jar from scratch.  The jars here should all exist.  Also - that param isn't available on the method I'm calling as I'm passing a URI, not an AbsolutePath